plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -284,11 +284,12 @@ static void tokudb_backup_error_fun(int error_number, const char *error_string,
 static bool tokudb_backup_check_slave_sql_thread_running(THD *thd) {
      scoped_lock_wrapper<BasicLockableMysqlMutextT>
         with_LOCK_active_mi_locked(
-        BasicLockableMysqlMutextT(&LOCK_active_mi));
+        (BasicLockableMysqlMutextT(LOCK_active_mi)));
```

Comment:
This is a good catch and I think a genuine bug that I am very surprised wasn't caught by compiler warning/error or a real user situation.